### DISCUSSION TITLE
使用Railway部署是不是没有办法使用语音识别功能？

### DISCUSSION THREAD

**Discussion by JeremiahLee11** [zhayujie/chatgpt-on-wechat#675] [Q&A] [answered]
我已经将 speech recognition 和 voice_reply_voice 这两个选项都改成 true了，但是微信那边还是只能通过文字来交流。
发送语音的话会显示：
raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'ThreadPoolExecutor-0_0'.
然后没有任何回复。
想请问一下是我设置有误还是Railway部署使用不了该功能？

**Comment by lanvent** [comment, marked as answer]
这是bug，已经修复好了，railway重新部署吧。
